openshift/cluster-prepare.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Prepare Cluster on Openshift CI
+# - Creates testing Namespace
+# - Setup Openshift Serverless and Openshift Pipelines
+# - Creates Test GitServer service
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+BASEDIR=$(dirname "$0")
+INSTALL_SERVERLESS="${INSTALL_SERVERLESS:-true}"
+INSTALL_PIPELINES="${INSTALL_PIPELINES:-true}"
+INSTALL_GITSERVER="${INSTALL_GITSERVER:-true}"
+GITSERVER_IMAGE="${GITSERVER_IMAGE:-ghcr.io/jrangelramos/gitserver-unpriv:latest}"
```

Comment:
Is the image multiarch? Probably does not have to be right now, still it would be nice.

Reply:
Just curious: who populates GITSERVER_IMAGE?

Reply:
Yea.. it is not multiach. and correct, unpriv if for unprivileged